src/Loading/index.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from "react";
+import styled, { keyframes } from "styled-components";
+
+import defaultTokens from "../defaultTokens";
+import TYPE_OPTIONS from "./consts";
+import type { Props } from "../Loading/index";
+import Spinner from "../icons/Spinner";
+
+// Animations
+const SpinnerAnimation = keyframes`
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+`;
+
+const LoaderAnimation = keyframes`
+  0% {opacity: .3; transform:translateY(0px);}
+  20% {opacity: 1; transform: translateY(-3px);}
+  40%  {opacity: .3; transform:translateY(0px);}
+  100%  {opacity: .3; transform:translateY(0px);}
+`;
+
+const LoadingContainer = styled(({ tokens, type, ...props }) => <div {...props} />)`
+  position: ${({ type }) => type === TYPE_OPTIONS.BUTTON_LOADER && "absolute"};
+  left: ${({ type }) => type === TYPE_OPTIONS.BUTTON_LOADER && "0"};
+  width: ${({ type }) => type === TYPE_OPTIONS.BUTTON_LOADER && "100%"};
+  height: ${({ tokens, type }) => tokens.heightLoadingContainer[type]};
+  padding: 0 ${({ theme }) => theme.orbit.spaceSmall};
+  display: flex;
+  flex-direction: ${({ type }) => (type === TYPE_OPTIONS.PAGE_LOADER ? "column" : "row")};
+  justify-content: ${({ tokens, type }) => tokens.alignLoadingContainer[type]};
+  align-items: center;
+  overflow: hidden;
+`;
+
+LoadingContainer.defaultProps = {
+  theme: defaultTokens,
+};
+
+const LoadingText = styled(({ theme, type, ...props }) => <div {...props} />)`
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextNormal};
+  color: ${({ theme }) => theme.orbit.paletteInkLighter};
+  line-height: ${({ theme }) => theme.orbit.lineHeightText};
+  margin-top: ${({ theme, type }) => type === TYPE_OPTIONS.PAGE_LOADER && theme.orbit.spaceLarge};
+  margin-left: ${({ theme, type }) => type !== TYPE_OPTIONS.PAGE_LOADER && theme.orbit.spaceSmall};
+`;
+
+LoadingText.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledSpinner = styled(Spinner)`
+  width: ${({ type }) => type !== TYPE_OPTIONS.BUTTON_LOADER && "45px"};
```

Comment:
Can we redefine sizes and spaces, to be orbit compliant? `45px` doesn't fit into our size/space system.